src/handlers/utils.js

```diff
@@ -38,15 +51,15 @@ export function createFlexMessageText(text = '') {
 export function createTypeWords(type) {
   switch (type) {
     case 'RUMOR':
-      return '含有不實訊息';
+      return t`Contains misinformation`;
     case 'NOT_RUMOR':
-      return '含有真實訊息';
+      return t`Contains true information`;
     case 'OPINIONATED':
-      return '含有個人意見';
+      return t`Contains personal perspective`;
     case 'NOT_ARTICLE':
-      return '不在查證範圍';
+      return t`Invalid request`;
   }
-  return '回應的狀態未定義！';
+  return 'Undefined';
```

Comment:
Should we apply i18n to this as well?

Reply:
I think it's ok, we don't need to add a translation entry for a content that only appears when we omit something.

When this appears, it means we are adding more cases and provide proper translation to that case.